### PR TITLE
Typescript

### PR DESCRIPTION
**Pull request by cope**
typescript

### CODE REVIEW

File: src/player.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	 * @param {number} rating - Rating
+	 * @returns {object} PrefPlayer instance
+	 */
+	constructor(username: string, rating: number, score: number) {
+		this._username = username;
+		this._rating = rating;
+		this._score = score;
+		this._change = 0;
+
+		return this;
```
Comment (dzoni77) [inline src/player.ts]: "return this" is not needed in the ts constructors - it always returns instance of the class, just like in Java and C#

File: src/rating.ts [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+		this._p2 = p2;
+		this._p3 = p3;
+		this._bula = bula;
+
+		let {change1, change2, change3} = _calculateChanges(p1, p2, p3, bula);
+		this._p1.change = change1;
+		this._p2.change = change2;
+		this._p3.change = change3;
+
+		return this;
```
Comment (dzoni77) [inline src/rating.ts]: Same as above, not needed in TS constructor

**Review by dzoni77 — APPROVED** [review]

File: src/player.ts (hunk 1 above)
Comment (cope) [reply src/player.ts]: Fixed.

File: src/rating.ts (hunk 2 above)
Comment (cope) [reply src/rating.ts]: Fixed

**Pull request merged**